README.md

```diff
@@ -94,7 +94,7 @@ Select the corresponding database name from the table above for the site you wis
 'CREATE DATABASE IF NOT EXISTS wordpress_api_jquery_com;' | docker exec -i jquerydb mysql -u root -proot
 ```
 
-Then, visit the local site in your browser to complete the WordPress installation, such as http://local.api.jquery.com/wp-admin/install.php. Make sure the address begins with `local.`.
+Then, visit the local site in your browser to complete the WordPress installation, such as http://local.api.jquery.com/wp-admin/install.php. Make sure the address begins with `local.`. Or visit https://local.api.jquery.com, if you created the SSL certificates.
```

Comment:
I consider this separate from step 8, which is to actually visit the homepage when either the "All users" or "Infrastructure team" path is taken (but not both). No need to mention the SSL certs yet. In fact, I find it could reduce confusion because `grunt deploy` has to use http. Then, once the database is set up in one of those two ways, visit the homepage on either http or https, depending on if certs are present.

Reply:
With the headings in place, I don't think step 8 is visible. It looks like an unrelated point at the end of the infra-member section. The addition of the wp-admin step introduces a reference to a URL that has to work earlier, so I figured maybe we need to mention this here. If we leave it out here, we assume the reader will know, which is perhaps fine, but then we don't need it anywhere. What do you prefer?

Reply:
Now that I read it again, I guess we're thinking the main reader is different. I think I'd be happy with 2 little changes:

1. Add "Team members can skip this" to the "Construct the database" header
2. Move mention of https to after `grunt deploy` below (in this same section).

Reply:
```suggestion
Then, visit the local site in your browser to complete the WordPress installation, such as http://local.api.jquery.com/wp-admin/install.php. Make sure the address begins with `local.`.
```

Reply:
Sounds good to me!